webapi/src/main/scala/org/knora/webapi/store/iiif/impl/IIIFServiceSipiImpl.scala

```diff
@@ -334,12 +332,10 @@ object IIIFServiceSipiImpl {
   val layer: ZLayer[AppConfig & JWTService, Nothing, IIIFService] =
     ZLayer.scoped {
       for {
-        config <- ZIO.service[AppConfig]
-        // _          <- ZIO.debug(config)
+        config     <- ZIO.service[AppConfig]
         jwtService <- ZIO.service[JWTService]
         // HINT: Scope does not work when used together with unsafeRun to
-        // bridge over to Akka. Need to change this as soon Akka is removed
-        // httpClient <- ZIO.acquireRelease(acquire(config))(release(_))
+        // bridge over to Akka. TODO Need to change this as soon as Akka is removed
```

Comment:
not sure how much value this TODO comment provides

Reply:
I removed it.